nestjs/src/courses/courses.controller.ts

```diff
@@ -37,6 +37,18 @@ export class CoursesController {
     return data.map(it => new CourseDto(it));
   }
 
+  @Post('/')
```

Comment:
If you have migrated an endpoint from Koa to Nest, please delete the old code or create an issue for its removal